web/src/features/charts/bar-breakdown/BarBreakdownEmissionsChart.tsx

```diff
@@ -26,7 +26,7 @@ interface BarBreakdownEmissionsChartProps {
   ) => void;
   onProductionRowMouseOut: () => void;
   onExchangeRowMouseOver: (
-    mode: ElectricityModeType,
+    mode: ZoneKey,
```

Comment:
Should the mode really be a ZoneKey (which is just a string)?

Reply:
Yes, but I have just made a new commit that renames it from "mode" to "zoneKey" for exchanges - it was quite confusing that it was called "mode" but always were a zoneKey :)